src/Containers/RightSideBar/EditImage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from '@Inputs/Button';
+
+// API KEY: AIzaSyBW6afsWt6KM6yvzXgfJ9UmUGImHvedZbc
+
+// STYLED COMPONENTS
```

Comment:
Same as above about unnecessary comments. Code should speak for itself in terms of clarity here